From 9aef04d9baa0a4bb5b8db92e9ab93b0a857a7659 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Thu, 9 Jan 2014 20:08:04 -0300
Subject: [PATCH] Makefile.org: enclose CC parameter passing in quotes

The compiler invocation might contain a space like when using ccache.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>

diff -Nura openssl-1.0.1f.orig/Makefile.org openssl-1.0.1f/Makefile.org
--- openssl-1.0.1f.orig/Makefile.org	2014-01-09 19:57:23.324040960 -0300
+++ openssl-1.0.1f/Makefile.org	2014-01-09 20:11:23.821070726 -0300
@@ -302,7 +302,7 @@
 			FIPSLD_CC="$(CC)"; CC=$(FIPSDIR)/bin/fipsld; \
 			export CC FIPSLD_CC FIPSLD_LIBCRYPTO; \
 		fi; \
-		$(MAKE) -e SHLIBDIRS=crypto  CC=$${CC:-$(CC)} build-shared; \
+		$(MAKE) -e SHLIBDIRS=crypto  CC="$${CC:-$(CC)}" build-shared; \
 		touch -c fips_premain_dso$(EXE_EXT); \
 	else \
 		echo "There's no support for shared libraries on this platform" >&2; \
